addons/point_of_sale/static/src/app/store/product_configurator_popup/product_configurator_popup.scss

```diff
@@ -0,0 +1,22 @@
+.configurator_color{
+    &.ptav-not-available {
+        opacity: 1;
+
+        &:after {
+            content: "";
+            @include o-position-absolute(-5px, -5px, -5px, -5px);
+            border: 2px solid map-get($theme-colors, 'danger');
+            border-radius: 50%;
+            background: str-replace(url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' width='39' height='39'><line y2='0' x2='39' y1='39' x1='0' style='stroke:#{map-get($theme-colors, 'danger')};stroke-width:2'/><line y2='1' x2='40' y1='40' x1='1' style='stroke:rgb(255,255,255);stroke-width:1'/></svg>"), "#", "%23") ;
+            background-position: center;
+            background-repeat: no-repeat;
```

Comment:
please reference the files were you copied the code from.
https://github.com/odoo/odoo/blob/2ffc1d16dc0fa45ef2bc4b27e814c7ca241fe9af/addons/sale/static/src/js/product_template_attribute_line/product_template_attribute_line.scss#L47-L60

Ideally we would just import that css file, but as pos does not depend on sale we can copy it for now.
Please add a comment explaining that the code is coming from the specific file in sales and that perhaps we'll find a way to avoid the duplication in the future

Reply:
@vlst-odoo 
Done